crow/recipes/binned_exact.py

```diff
@@ -60,45 +60,19 @@ def _setup_with_completeness(self):
             self._completeness_distribution = self.completeness.distribution
 
     def _setup_with_purity(self):
-        """Makes mass distribution use additional integral with completeness"""
-        if self.purity is None:
+        """Makes mass distribution use additional integral with purity"""
+        if self.purity is None: 
```

Comment:
I think this is good. Because if there is purity, we have to change both the mass distribution and the purity function. I just don't think it is a good name to say then that the mass distribtuion impure is mass_dist * purity. But just a naming problem